### ISSUE TITLE
Add ROAS plot

### ISSUE THREAD

**Issue by lucianopaz** [pymc-labs/pymc-marketing#72]
Add a method to compute the return of ad spend for certain channels and plot it. The plot could follow the style of [figure 3 from Jin et al 2017](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/46001.pdf), which I'll copy down here just as a reference.

![image](https://user-images.githubusercontent.com/5230109/202191252-9bb6f253-ebd7-4ab8-8d5f-d1a02b7aebc7.png)

**Comment by ricardoV94** [comment]
Challenge: We need to do a forward pass to get these results.

**Comment by wd60622** [comment]
Could the "channel_contributions" Deterministic be used for this? 

For instance, 

```python
# (chain, draw, date, channel)
mmm.fit_result["channel_contribution"] / mmm.l
```

Have any thoughts on the difficulty of this @juanitorduz?

**Comment by juanitorduz** [comment]
Yes! We just need to be careful with the adstock contribution (similar as you did with the out-of-sample feature ;) )

**Comment by wd60622** [comment]
>  Yes! We just need to be careful with the adstock contribution (similar as you did with the out-of-sample feature ;) )

Would this affect more than the first `l_max` time periods?

**Comment by anevolbap** [comment]
This looks resolved on main: `mmm.plot_interactive.roas()` plots per-channel ROAS with HDI intervals ([`pymc_marketing/mmm/plot_interactive.py`](https://github.com/pymc-labs/pymc-marketing/blob/ea7cec6f0c126fbaa5c3483b3d5b6501a56b7572/pymc_marketing/mmm/plot_interactive.py#L832), added in #2259), and `mmm.incrementality.contribution_over_spend()` computes the values with the adstock carryover included (#2333). OK to close?
